### PR TITLE
 Add dependencies installation instructions and how to build

### PR DESCRIPTION
**Pull request by andrea-manzi**
# Summary 
 Add dependencies installation instructions and how to build

----

 #19

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -11,13 +11,33 @@ Sources to build [EGI documentation static pages](https://egi-foundation.github.
...
 - hugo
 - NodeJS
   - postcss-cli
   - autoprofixer
 
+### Installation and build
 
-The `.travis.yml` can be used as an example to see how to build the documentation.
+To install npm+nodejs please check the instructions at 
+
```
Comment (gwarf) [inline README.md]:
```suggestion
```

File: README.md [hunk 2]
```diff
@@ -11,13 +11,33 @@ Sources to build [EGI documentation static pages](https://egi-foundation.github.
...
+https://www.npmjs.com/get-npm
+
+The rest of the tools can be installed as follows:
+
+```console
+gem install mdl
+npm install postcss-cli
+npm install autoprefixer
+brew install hugo
```
Comment (gwarf) [inline README.md]: So we had various problems due to the hugo version, so now we include it into the repository, it's probably better to use this version as described in the testing locally section. In fact it maybe be better to merge or link those two sections together.

Comment (andrea-manzi) [reply README.md]: please check the new version:-)

File: README.md [hunk 3]
```diff
@@ -29,16 +29,25 @@ The rest of the tools can be installed as follows:
...
 ```
+The supported Hugo version packages are available under the `binaries` folder.
+
 
 To build the site, from the repository root
 
 ```console
 mdl -s relaxed -s style.rb -r ~MD002,~MD024 content/
-hugo
+./binaries/<platform>/hugo
```
Comment (gwarf) [inline README.md]:
```suggestion
./binaries/<platform>/hugo
```

File: README.md [hunk 4]
```diff
@@ -29,16 +29,25 @@ The rest of the tools can be installed as follows:
 gem install mdl
 npm install postcss-cli
 npm install autoprefixer
-brew install hugo
 ```
+The supported Hugo version packages are available under the `binaries` folder.
+
```
Comment (gwarf) [inline README.md]:
```suggestion
```

File: README.md [hunk 5]
```diff
@@ -17,9 +17,9 @@ Sources to build [EGI documentation static pages](https://egi-foundation.github.
   - postcss-cli
   - autoprofixer
 
-### Installation and build
+### Local installation and build
```
Comment (gwarf) [inline README.md]:
```suggestion
### Installing dependencies, building and testing
```

**Comment by andrea-manzi** [comment]
thanks, I applied your changes!

**Pull request merged**